CHANGELOG.md

```diff
@@ -72,6 +72,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Fixed
 
 - Correct the release date of `0.20.0` to `2024-04-25`.
+- Fixed `Int` type precision error that occurs when graphql API have instances
+  that are greater than `i32` (`u32`,`u64`,`isize`,`usize`,`i64`) for
+  `NetworkThreat`,`AccountTotalCount`,`ClusterTotalCount`,`ColumnTimeSeries`,
+  `ModelIndicator`.
```

Comment:
@marioCluml @henry0715-dev 

The previously merged commits documented this in the section `Changed`, but this did in the section `Fixed`. If we consider this a bug we need to put all into `Fixed`, but I think it is still Ok to keep it in the `Changed`.  Either way, we have to make it consistent.

Because the correction of README can be done right before releasing a new version, I am merging this as it is. And, I will take care of the README later.

Reply:
Yes because we were merging it sequentially I was going to rebase this branch on top of main after part 2 was merged and in doing so fix the CHANGELOG that has part 2 CHANGELOG entry. Then for part 4 I was going to do the same process as well. So that way it will be more consistent while being merged sequentially.

Reply:
I'll be back home in about an hour or so and can fix it then since part 2 was merged earlier today. I was waiting for the part 2 to be merged before fixing this

Reply:
@marioCluml 

I suggest fixing the README in #270 instead of this. By doing so, I think we will not need to enumerate all the items and would rather mention like "all the relevant APIs" if we cover all.